docs/includes/apiargs-MongoDBClient-method-construct-param.yaml

```diff
@@ -2,46 +2,41 @@ arg_name: param
 name: $uri
 type: string
 description: |
-  The URI of the standalone, replica set, or sharded cluster to which to connect. Refer
-  to the :manual:`MongoDB connection string reference </reference/connection-string>`
-  for formatting.
-  
-  Defaults to ``mongodb://localhost:27017`` if unspecified.
+  The URI of the standalone, replica set, or sharded cluster to which to
+  connect. Refer to the :manual:`MongoDB connection string reference
+  </reference/connection-string>` for formatting.
+
+  Defaults to ``"mongodb://127.0.0.1:27017"`` if unspecified.
 interface: phpmethod
 operation: MongoDB\\Client::__construct
 optional: true
-position: 1
```

Comment:
I found that `position` was not required by the model. By default, the order is derived from the YAML file's own ordering (starting with 1 and incrementing with each new definition).